### ISSUE TITLE
Media library items media field left empty for images

### ISSUE THREAD

**Issue by nixman2** [JeffDarchuk/SitecoreSidekick#78]
First off, thanks for the great support on the last ticket I created.

I've got another one, which involves the media library items.

When I select a new folder with 1 image in it, it takes more than 20 minutes to sync the folder to another environment.
AND the new item won't have it's media field properly filled.

elapsed time:  1290.2705965999999 seconds

Some information which might be important is:
- the item is created in the dutch language.
- the item is of template /sitecore/templates/System/Media/Versioned/Jpeg
- if I uncheck every option beneath Sitecore Advanced Options, it takes 1 second tops to sync, and the media field is correctly filled. However, I can't tell the content people to always uncheck these options if they are syncing media items.

So two problems really, but I'm really hoping to get a bugfix for the media field that's left empty when
 Run using the event disabler.
 Run using bulk update context.
 Use Item Blaster
 
 are checked.

**Comment by JeffDarchuk** [comment]
People have reported errors similar to this related to the Item Blaster support.  Does it work correctly if you only run the operation without Item Blaster?

**Comment by nixman2** [comment]
Yes I can confirm it does work correctly if I only uncheck use Item blaster

**Comment by JeffDarchuk** [comment]
Ah, unfortunately it's a common problem that some people have.  In future versions I'm planning to have it disabled by default and have the main use case for item blaster be if you had an environment that needed a massive amount of creates and didn't want to wait for Sitecore's item installation for each one.

**Comment by nixman2** [comment]
so somewhere in the content migrator code in a configuration file I did see an option called<DefaultUseItemBlaster>

can we simple add <DefaultUseItemBlaster>false</DefaultUseItemBlaster> to our configuration files to fix it?

**Comment by JeffDarchuk** [comment]
There isn't a way to completely disable it, but in the alpha versions there's a default state configuration that you can use to have the checkbox be disabled by default to avoid these sorts of problems while still having it available if you need to move a massive amount of content.

**Comment by nlounds** [comment]
I think I'm encountering this same issue, but it doesn't matter which boxes I check or uncheck (I think I've tried all combos), the media field is always blank.  My co-worker was able to get content from another site to eventually work after repeated attempts, but this new use case is failing every single time.

Also, even though the remote site has the media attached and the local on does not, the tool sees them as being identical.  So to re-attempt the copy over, I have to delete the item(s) first.

Any ideas on what I could try?
